package.json

```diff
@@ -123,7 +123,7 @@
     "@graphql-codegen/typescript-react-apollo": "^3.2.5",
     "@percy/cli": "^1.21.0",
     "@percy/cypress": "^3.1.2",
-    "@saleor/app-sdk": "0.41.0",
+    "@saleor/app-sdk": "0.0.0-pr-20230721063335",
```

Comment:
**question**: is this PR will WIP? 🤔

Reply:
Yes and no. I need to have a review to verify if this approach is good. Once it happens I can release SDK to "latest" version.

I don't want to release SDK before this is merged, because if we have API changes due to this PR, I will have breaking changes.

So once this is approved I will replace an SDK version to 0.42.0